disposable_email_blacklist.conf

```diff
@@ -310,7 +311,6 @@ antispam.de
 antispam24.de
 antispammail.de
 anyalias.com
-aoeuhtns.com
```

Comment:
why is this domain being removed?

Reply:
my error when adding other domains fixed